docs/modules/ROOT/pages/payment-methods.adoc

```diff
@@ -1,5 +1,13 @@
-= Payment Methods
+= Managing Payment Methods
 
-Hazelcast Cloud works with Stripe Inc for safely processing payments. We never keep any credit card data in our database. You can add your credit card details under *Account* > *Payment Methods* menu as below:
+Hazelcast Cloud works with Stripe Inc for safely processing payments. We never keep any credit card data in our database.
+
+== Prerequisites
+
+To manage payment methods for a team, you must have either an admin or finance role. See xref:teams-and-users.adoc[].
```

Comment:
I see `.adoc` url in the referred link text, is that valid usage? We dont need to show adoc extension to end user
![Screen Shot 2021-09-02 at 15 24 19](https://user-images.githubusercontent.com/1237982/131843062-4a59eae5-3375-4cd3-820e-091889563ef6.png)

Reply:
All of these .adoc links were broken. They are fixed now 😄